src/model/platform-setup/setup-mac.ts

```diff
@@ -30,13 +30,41 @@ class SetupMac {
     }
   }
 
+  private static getModuleParametersForTargetPlatform(targetPlatform: string): string {
+    let command = '';
+    switch (targetPlatform) {
+      case 'iOS':
+        command += '--module ios ';
+        break;
+      case 'tvOS':
+        command += '--module tvos ';
+        break;
+      case 'Android':
+        command += '--module android ';
+        break;
+      case 'MacOS':
```

Comment:
this case needs to support `StandaloneOSX` MacOS isnt a valid targetPlatform for unity. see values here https://docs.unity3d.com/ScriptReference/BuildTarget.html